[2025-04-02T16:10:15.871+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: People_Data_Pipeline.ensure_tmp_dir scheduled__2025-04-01T00:00:00+00:00 [queued]>
[2025-04-02T16:10:15.882+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: People_Data_Pipeline.ensure_tmp_dir scheduled__2025-04-01T00:00:00+00:00 [queued]>
[2025-04-02T16:10:15.883+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-04-02T16:10:15.900+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ensure_tmp_dir> on 2025-04-01 00:00:00+00:00
[2025-04-02T16:10:15.906+0000] {standard_task_runner.py:60} INFO - Started process 262 to run task
[2025-04-02T16:10:15.910+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'People_Data_Pipeline', 'ensure_tmp_dir', 'scheduled__2025-04-01T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/people_dag.py', '--cfg-path', '/tmp/tmp1f6exjdv']
[2025-04-02T16:10:15.913+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask ensure_tmp_dir
[2025-04-02T16:10:15.981+0000] {task_command.py:423} INFO - Running <TaskInstance: People_Data_Pipeline.ensure_tmp_dir scheduled__2025-04-01T00:00:00+00:00 [running]> on host 4c976e0a7b20
[2025-04-02T16:10:16.088+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='People_Data_Pipeline' AIRFLOW_CTX_TASK_ID='ensure_tmp_dir' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-01T00:00:00+00:00'
[2025-04-02T16:10:16.090+0000] {people_dag.py:120} INFO - Temporary directory already exists: /tmp
[2025-04-02T16:10:16.091+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-02T16:10:16.104+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=People_Data_Pipeline, task_id=ensure_tmp_dir, execution_date=20250401T000000, start_date=20250402T161015, end_date=20250402T161016
[2025-04-02T16:10:16.124+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-02T16:10:16.154+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-02T16:31:16.961+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: People_Data_Pipeline.ensure_tmp_dir scheduled__2025-04-01T00:00:00+00:00 [queued]>
[2025-04-02T16:31:16.972+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: People_Data_Pipeline.ensure_tmp_dir scheduled__2025-04-01T00:00:00+00:00 [queued]>
[2025-04-02T16:31:16.972+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-04-02T16:31:16.988+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ensure_tmp_dir> on 2025-04-01 00:00:00+00:00
[2025-04-02T16:31:16.992+0000] {standard_task_runner.py:60} INFO - Started process 240 to run task
[2025-04-02T16:31:16.997+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'People_Data_Pipeline', 'ensure_tmp_dir', 'scheduled__2025-04-01T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/people_dag.py', '--cfg-path', '/tmp/tmp04cy00ol']
[2025-04-02T16:31:17.000+0000] {standard_task_runner.py:88} INFO - Job 2: Subtask ensure_tmp_dir
[2025-04-02T16:31:17.063+0000] {task_command.py:423} INFO - Running <TaskInstance: People_Data_Pipeline.ensure_tmp_dir scheduled__2025-04-01T00:00:00+00:00 [running]> on host e3653afa66c4
[2025-04-02T16:31:17.161+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='People_Data_Pipeline' AIRFLOW_CTX_TASK_ID='ensure_tmp_dir' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-01T00:00:00+00:00'
[2025-04-02T16:31:17.163+0000] {people_dag.py:114} INFO - Temporary directory already exists: /tmp
[2025-04-02T16:31:17.164+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-02T16:31:17.174+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=People_Data_Pipeline, task_id=ensure_tmp_dir, execution_date=20250401T000000, start_date=20250402T163116, end_date=20250402T163117
[2025-04-02T16:31:17.250+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-02T16:31:17.282+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
